src/pages/Booking/Booking.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import Calendar from './components/Calendar/Calendar';
+import Movie from './components/Movie/Movie';
+
+// [컴포넌트 분리]
+// 1. 달력(월, 달력 아이콘)
+// 2. 영화
+// 3. 극장
+// 4. 시간
+
+function Booking() {
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```
Fragment 적용 가능한지 한번 검토해주세요